### PR TITLE
Starting server and config reloading

### PR DESCRIPTION
**Pull request by tvh**

### CODE REVIEW

File: src/tokenserver.hs [hunk 1]
```diff
@@ -1,6 +1,60 @@
...
+    args <- getArgs
+    let confFile = case args of
+            [] -> "/etc/anchor-token-server/anchor-token-server.conf"
+            [fp] -> fp
+            _ -> error "Usage: anchor-token-server [CONFIG]"
+    (conf', _) <- autoReloadGroups autoConfig [("main.",Required confFile)]
+    let conf = subconfig "main" conf'
+    active <- newTVarIO True
+    srv <- newEmptyTMVarIO
+    restartServer active srv conf
```
Comment (tranma) [inline src/tokenserver.hs]: so, `active` indicates whether the thread wants to enter its critical section, and `srv` indicates whether it is currently in its critical section?

Comment (thsutton) [inline src/tokenserver.hs]: `active` is set by its [FitBit](https://www.fitbit.com/) based on how much it has walked today.

**Comment by tvh** [comment]
I removed the config reloading @thsutton. Should be good to merge as is.

**Comment by tranma** [comment]
looks fine

**Pull request merged**